python/taichi/main.py

```diff
@@ -32,10 +31,12 @@ def test_python(args):
     else:
         # run all the tests
         pytest_args = [test_dir]
-    if verbose:
+    if args.verbose:
         pytest_args += ['-s', '-v']
+    if args.rerun:
+        pytest_args += ['--reruns', '2']
```

Comment:
I'm fine with this approach, just wondering why not simply let users decide how many times they want to rerun?

Reply:
I thought a user will never want to rerun more than 3 times since already proof the test to be wrong and keep rerun only wasting time to get the result.

Reply:
Anyway, it doesn't matter at all, let's just use `-r2`.